frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss

```diff
@@ -239,7 +239,7 @@
 
             &:not([aria-disabled='true']):hover,
             &.LemonButton--active {
-                background-color: var(--bg-3000);
+                background-color: var(--glass-border-3000);
```

Comment:
At some point we probably want to change the name of that var

Reply:
I did consider that. The breadcrumbs and this button are the only places it's used right now so should be relatively easy to find a suitable name once we get a sense of where else it is needed